src/components/settings/SettingsLMSTab/ExistingCard.jsx

```diff
@@ -112,13 +121,23 @@ const ExistingCard = ({
     }
   };
 
+  const getLastSync = () => {
+    if (config.lastSyncErroredAt != null) {
+      const timeStamp = getTimeAgo(config.lastSyncErroredAt);
+      return <>Recent sync error:&nbsp; {timeStamp}<Icon className="small-icon text-danger-500" src={Error} /></>;
+    } if (config.lastSyncAttemptedAt != null) {
```

Comment:
nit: Don't strictly need this right now for flow reasons, but if we ever decide to make this function have only one return state in the future, can help avoid a bug.
```suggestion
    } else if (config.lastSyncAttemptedAt != null) {
```

Reply:
The only thing is that with a return, the else isn't necessary, because if it falls into that, it means it didn't return from the previous if